examples/kind/README.md

```diff
@@ -0,0 +1,72 @@
+# KIND Demo
+
+[kind](https://github.com/kubernetes-sigs/kind)(Kubernetes in Docker) is a tool for running local Kubernetes clusters using Docker container “nodes”. Azure CSI Driver will work in kind using Service Principal.
```

Comment:
nit: s/Azure CSI Driver/Azure Key Vault Provider for Secrets Store CSI Driver

Reply:
Done